src/datadog_directive.h

```diff
@@ -72,11 +72,39 @@ constexpr auto generate_directives(const T &...directives) {
   return merge_directives(directives...);
 }
 
+#define IGNORE_COMMAND(NAME, TYPE) \
+  { NAME, TYPE, silently_ignore_command, NGX_HTTP_LOC_CONF_OFFSET, 0, nullptr }
+
+#define ALIAS_COMMAND(SRC_DIR, TARGET_DIR, TYPE)                    \
```

Comment:
For me, src_dir evokes src_directory so I was a bit lost. To improve readability, I would argue that it would be better to fully write SRC_DIRECTIVE.

It is a bit more of a nit comment but similarly, when I see ALIAS_COMMAND("datadog_trust_incoming_span", "opentracing_trust_incoming_span"...), it evokes me that when we use datadog_trust we want to use opentracing while it is more the reverse

Reply:
Addressed in 79ebeff405491b385fce2e63b4b35ab0b409d58e.

> when I see ALIAS_COMMAND("datadog_trust_incoming_span", "opentracing_trust_incoming_span"...), it evokes me that when we use datadog_trust we want to use opentracing while it is more the reverse

This is a subjective take. At the end of the day, somebody will not be happy about. I guess I could use `constexpr` function with designated initializers. If that can help, it uses the same order than all posix command (`mv`, `cp` ...)